modules/common/src/main/java/unnamed/mmo/server/instance/TickTrackingInstance.java

```diff
@@ -0,0 +1,34 @@
+package unnamed.mmo.server.instance;
+
+import net.minestom.server.instance.IChunkLoader;
+import net.minestom.server.instance.InstanceContainer;
+import net.minestom.server.world.DimensionType;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+public class TickTrackingInstance extends InstanceContainer {
+    //todo i (matt) really dont think this should be in common, but not sure where
```

Comment:
This is fine for now, but I am slightly concerned that people will forget to create this custom instance container and instead create the default one, so this may not be the best way to go about it

May need some sort of globally accessible tick counter.

Reply:
Noted in Discord, but long story short I agree with this comment.